package.json

```diff
@@ -26,6 +26,7 @@
   "dependencies": {
     "@babel/runtime": "^7.9.2",
     "babel-plugin-fbt-runtime": "^0.9.4",
+    "cls-hooked": "^4.2.2",
```

Comment:
Used cls-hooked instead of node-continuation-local-storage:
See https://github.com/othiym23/node-continuation-local-storage/issues/98#issuecomment-317847871 for more info.

There's maybe hopes to find something more elegant with this other Express plugin: https://github.com/othiym23/node-continuation-local-storage/issues/98#issuecomment-332829028